aiidalab/app.py

```diff
@@ -569,15 +569,15 @@ def install(
                         "Performing rollback to previously installed version."
                     )
                     self._restore_from(trash_path)
-            except RuntimeError as inner_error:
-                logger.error(f"Rollback failed due to error: {inner_error}!")
+            except RuntimeError:
+                logger.exception("Rollback failed! Consider re-installing the app.")
```

Comment:
Apparently, the exception is always automatically included when using `logger.exception` inside except statement.